app/models/profile.rb

```diff
@@ -173,6 +173,15 @@ def in_person_verification_pending?
     in_person_verification_pending_at.present?
   end
 
+  def deactivate_due_to_gpo_expiration
```

Comment:
Given the name of this method, should we also set a deactivation reason?

Reply:
No, the intention is to move away from the use of deactivation reason.